api/types/provisioning.go

```diff
@@ -862,3 +876,31 @@ func (a *ProvisionTokenSpecV2TerraformCloud) checkAndSetDefaults() error {
 
 	return nil
 }
+
+func (a *ProvisionTokenSpecV2Bitbucket) checkAndSetDefaults() error {
```

Comment:
Would be nice to include tests in provisioning_test.go for this.

Reply:
Ah, good call, missed that file. Added!